index.js

```diff
@@ -37,8 +37,12 @@ const storeKey = "store";
 const app = Elm.Main.init({
   flags: {
     clientUrl: location.origin + location.pathname,
-    enableFoodSection: process.env.ENABLE_FOOD_SECTION === "True",
-    enableObjectSection: process.env.ENABLE_OBJECT_SECTION === "True",
+    enabledSections: {
+      food: process.env.ENABLE_FOOD_SECTION === "True",
+      objects: process.env.ENABLE_OBJECT_SECTION === "True",
```

Comment:
Maybe we should use `object` or `objects` everywhere but not mixing and matching both?